src/main/java/hudson/remoting/SynchronousCommandTransport.java

```diff
@@ -57,14 +57,14 @@ public ReaderThread(CommandReceiver receiver) {
         public void run() {
             final String name =channel.getName();
             try {
-                while(!channel.isInClosed()) {
+                while (!channel.isInClosed()) {
```

Comment:
These whitespace changes were enforced by IntelliJ when I made an unrelated debugging change (which is not included in this PR).